docs/3.0. Behaviour.md

```diff
@@ -22,11 +22,21 @@ The API provides a mechanism to change the settings associated with logical Sour
 
 ## Senders
 
+`/senders` returns all Senders on the Node which is associated with the API.
+
```

Comment:
I think the Senders and Receivers should both have one sentence saying "these GUIDs and resource correspond with what is exposed with IS-04". It's the same for both.

```suggestion
```

Reply:
@prince-chrismc, do you mean in Behaviour or it's enough if only in RAML?

Reply:
Scanning quickly I did not see this in the RAML, regardless I think the Behavior section here is enough

Reply:
I fixed it in f78576af69c869e5cf06f64c284c63a55ab5ecb8.